rbac/rbac/settings.py

```diff
@@ -506,6 +506,7 @@
 WORKSPACE_RESOURCE_TYPE = ENVIRONMENT.get_value("WORKSPACE_RESOURCE_TYPE", default="groups")
 WORKSPACE_ATTRIBUTE_FILTER = ENVIRONMENT.get_value("WORKSPACE_ATTRIBUTE_FILTER", default="group.id")
 WORKSPACE_HIERARCHY_ENABLED = ENVIRONMENT.bool("WORKSPACE_HIERARCHY_ENABLED", False)
+WORKSPACE_ORG_CREATION_LIMIT = ENVIRONMENT.get_value("WORKSPACE_ORG_CREATION_LIMIT", default=10)
```

Comment:
let's set this to something bigger for example 3000, (we have 2203 in stage for some groups )